WooCommerce/Classes/ViewModels/Order List/OrderListCellViewModel.swift

```diff
@@ -0,0 +1,74 @@
+import Foundation
+import Yosemite
+
+struct OrderListCellViewModel {
```

Comment:
⚠️ Would you mind moving this to the same folder as `OrderTableViewCell`? We're trying to organize files by feature now instead of grouping all ViewModels together. Same for the `OrderListCellViewModelTests.swift` too please. 🙂 

<img src="https://user-images.githubusercontent.com/198826/102619363-907bd100-40f9-11eb-8b39-39ee150c91bd.png" width="300">


P.S. `ViewRelated` is a weird folder name. I think we should get rid of that soon. 😬

Reply:
Sorry I wasn't aware of this. Moved it! 👍

Reply:
Thanks!